backend/src/main/scala/cromwell/backend/BackendJobExecutionActor.scala

```diff
@@ -15,8 +15,8 @@ object BackendJobExecutionActor {
 
   // Commands
   sealed trait BackendJobExecutionActorCommand extends BackendWorkflowLifecycleActorCommand
-  case object ExecuteJobCommand extends BackendJobExecutionActorCommand
-  case object RecoverJobCommand extends BackendJobExecutionActorCommand
+  case object ExecuteJobCommand extends BackendJobExecutionActorCommand { def isExecuteJobCommand(x: BackendJobExecutionActorCommand) = x == ExecuteJobCommand }
```

Comment:
I couldn't find the usage for isExecute/RecoverJobCommand, or this was added for tests or something?